docs/modules/user-guides/pages/use-in-memory-db-to-manage-test-data.adoc

```diff
@@ -113,5 +113,5 @@ UPDATE test.students SET firstname='Mary' WHERE lastname='Miller';
 When I execute SQL query `
 SELECT * FROM test.students
 ` against `testh2db` and save result to scenario variable `h2-records`
-Then `${h2-records}` is equal to `[{firstname=Mary, lastname=Miller}, {firstname=Alex, lastname=Magnus}]`
+Then `${h2-records}` is equal to `[{FIRSTNAME=Mary, LASTNAME=Miller}, {FIRSTNAME=Alex, LASTNAME=Magnus}]`
```

Comment:
is it required change?

Reply:
Not really related to the issue.
When I studied H2 and ran scenarios, one of them failed:
<details>
<img width="1792" alt="Screenshot 2022-07-27 at 15 16 59" src="https://user-images.githubusercontent.com/66934999/181234502-11e09312-d9a6-43fa-bf77-5d3a475b251f.png">
</details>